### DISCUSSION TITLE
python-language-server executable is pyls now instead of pylsp

### DISCUSSION THREAD

**Discussion by strizhkindenis** [helix-editor/helix#8854] [General] [answered]
### Summary

If you install latest `python-language-server` from `pip` there is no pylsp executable only pyls.

### Reproduction Steps

`pip install 'python-language-server[all]'`

expected: python language server works

happened:
```console
Configured language servers:
  ✘ pylsp: 'pylsp' not found in $PATH
```

fix: symlink `pyls` -> `pylsp`

### Platform

Linux

### Terminal Emulator

foot version: 1.16.2

### Installation Method

emerge (gentoo's package manager)

### Helix Version

helix 23.10 (f6021dd0)

**Comment by kirawi** [comment, marked as answer]
They are different LSPs. https://github.com/helix-editor/helix/wiki/How-to-install-the-default-language-servers#python---pylsp

**Reply by strizhkindenis** [reply]
Yeah, got them mixed up. Thank you.
